modules/virt-adding-secret-configmap-service-account-to-vm.adoc

```diff
@@ -0,0 +1,45 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-managing-configmaps-secrets-service-accounts.adoc
+
+[id="virt-adding-secret-configmap-service-account-to-vm_{context}"]
+
+= Adding a secret, ConfigMap, or service account to a virtual machine
+
+Add a secret, ConfigMap, or service account to a virtual machine by using the {product-title} web console.
+
+.Prerequisites
+
+* You must have at least one available secret, ConfigMap, or service account
```

Comment:
I think OCP has a bunch of these by default. And it looks like new projects get some default SAs and secrets at creation. It might help to define this a bit better. 
Also, not sure about 'available'. What would make it unavailable? I can add the same SA/secret/ConfigMap to multiple VMs. It seems as though they just have to exist in my namespace. (Do they *have* to be in the same namespace as the VM to be attached to them?) 
@jelkosz ^?

Reply:
indeed, they only have to exist in your namespace (and they have to be in the same namespace as your VM is). Maybe the prerequisite could only contain something like "make sure the secret, ConfigMap, or service account exist in the same namespace than the VM you want to add it to" or something like that.